src/Synology.Api.Client/SynologyHttpClient.cs

```diff
@@ -93,19 +93,19 @@ private async Task<T> HandleSynologyResponse<T>(HttpResponseMessage httpResponse
                 case HttpStatusCode.OK:
                     {
                         var response = await httpResponse.Content.ReadFromJsonAsync<ApiResponse<T>>();
-                        if (!response.Success)
+                        if (!response?.Success ?? true)
```

Comment:
Maybe we change this line to make it easier to understand? Something like `if (!(response?.Success ?? false))`